toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -19,6 +27,16 @@ func doCustomizations(baseConfigPath string, config *imagecustomizerapi.SystemCo
 		return err
 	}
 
+	err = runScripts(baseConfigPath, config.PostInstallScripts, imageChroot)
+	if err != nil {
+		return err
+	}
+
+	err = runScripts(baseConfigPath, config.FinalizeImageScripts, imageChroot)
```

Comment:
Might be a good idea to add a comment here that this should be kept as the final step.